### PR TITLE
fix(alert): Ignore ipmi Entity Presence type alert

### PR DESCRIPTION
**Pull request by jneo8**
Alert rule ipmi_generic_sensor_value ignoe type="Entity Presence"

Fix: #102

### CODE REVIEW

File: src/prometheus_alert_rules/ipmi_sensors.yaml [hunk 1]
```diff
@@ -75,13 +75,13 @@ groups:
...
-      expr: ipmi_generic_sensor_value{state=~"Warning|Critical"}
+      expr: ipmi_generic_sensor_value{state=~"Warning|Critical", type!="Entity Presence"}
       for: 0m
       labels: 
         severity: "{{ toLower $labels.state }}"
       annotations:
         summary: IPMI sensor value in {{ toLower $labels.state }} state. (instance {{ $labels.instance }})
         description: |
-          A sensor value, recorded by ipmi sensor, in {{ toLower $labels.state }} state.
+          A sensor value, recorded by ipmi sensor, in {{ toLower $labels.state }} state and ignore type Entity Presence.
```
Comment (aieri) [inline src/prometheus_alert_rules/ipmi_sensors.yaml:85]:
```suggestion
          A sensor value, recorded by ipmi sensor, in {{ toLower $labels.state }} state. Entity Presence sensors are ignored.
```

File: tests/unit/test_alert_rules/test_ipmi_sensors.yaml [hunk 2]
```diff
@@ -226,7 +228,7 @@ tests:
             exp_annotations:
               summary: IPMI sensor value in warning state. (instance ubuntu-11)
               description: |
-                A sensor value, recorded by ipmi sensor, in warning state.
+                A sensor value, recorded by ipmi sensor, in warning state and ignore type Entity Presence.
```
Comment (aieri) [inline tests/unit/test_alert_rules/test_ipmi_sensors.yaml:231]:
```suggestion
                A sensor value, recorded by ipmi sensor, in warning state. Entity Presence sensors are ignored.
```

File: tests/unit/test_alert_rules/test_ipmi_sensors.yaml [hunk 3]
```diff
@@ -236,6 +238,6 @@ tests:
             exp_annotations:
               summary: IPMI sensor value in critical state. (instance ubuntu-12)
               description: |
-                A sensor value, recorded by ipmi sensor, in critical state.
+                A sensor value, recorded by ipmi sensor, in critical state and ignore type Entity Presence.
```
Comment (aieri) [inline tests/unit/test_alert_rules/test_ipmi_sensors.yaml:241]:
```suggestion
                A sensor value, recorded by ipmi sensor, in critical state. Entity Presence sensors are ignored.
```

**Review by aieri — CHANGES_REQUESTED** [review]
Some minor rewording. Also please add context to this change and the related bug. Why are we doing this? Is this related to https://bugs.launchpad.net/charm-hw-health/+bug/1891915 ?

**Review by Pjack — APPROVED** [review]

**Comment by jneo8** [comment]
> Is this related to https://bugs.launchpad.net/charm-hw-health/+bug/1891915 ?

It's a different case that want to ignore Entity Absent. I would like to put it into another issue.

**Comment by jneo8** [comment]
Hi @aieri , I update the wording and put some information on the issue.

**Review by aieri — APPROVED** [review]

**Pull request merged**